dev/ansible/01_generate_secrets_vault.yml

```diff
@@ -0,0 +1,29 @@
+- name: Generate Secrets with Vault
+  shell: |
+      vault kv put secret/postgres \
+      username=$(vault write -field='value' gen/password length=12 symbols=0) \
+      password=$(vault write -field='value' gen/password length=12 symbols=0)
+  run_once: true
+  environment:
+      VAULT_ADDR: "http://127.0.0.1:8200"
```

Comment:
Do we actually need this? `VAULT_ADDR` is already set on the machine 
![image](https://user-images.githubusercontent.com/51820995/93851189-f948de00-fcaf-11ea-92d0-40ad4f1b0346.png)

Reply:
Hm, okay, fair enough!